src/helpers/offlineSyncModal.js

```diff
@@ -78,20 +85,23 @@ function offlineSyncModalFactory(proxy) {
             dialogShortcut.enable();
         })
         .on('destroy', () => {
+            proxy.off('reconnect.waiting');
+            proxy.off('disconnect.waiting');
             globalShortcut.enable();
             dialogShortcut.disable();
             dialogShortcut.clear();
         })
         .on('wait', () => {
+            hider.show(betweenButtonTextSelector);
```

Comment:
Still the same issue :)
Should be rather something like: `hider.show(waitingDialog.dialog.getDom().find(betweenButtonTextSelector));`
But a shorter code should be better ;)